modules/DOMStateStorage.js

```diff
@@ -20,6 +20,17 @@ export const saveState = (key, state) => {
       window.sessionStorage.setItem(createKey(key), JSON.stringify(state))
     }
   } catch (error) {
+    if (!window.sessionStorage) {
```

Comment:
Check this before attempting to write... no point in catching this as an exception